renderer/featuredrelations.go

```diff
@@ -2,9 +2,9 @@ package renderer
 
 import (
 	"fmt"
-
 	"github.com/anyproto/anytype-heart/util/pbtypes"
 	"github.com/gogo/protobuf/types"
+	"github.com/ipfs/go-cid"
```

Comment:
can't find where go-cid is used..?

Reply:
https://github.com/anyproto/anytype-publish-renderer/pull/110/files#diff-92b005dfdf171a12e2f9dd5a1f08858432f699c8de4a943654ddf34ed7121f32R77